CODEOWNERS

```
@@ -1,6 +1,7 @@
 *                @istio/wg-docs-maintainers-infra
 /.spelling       @istio/wg-docs-maintainers-english
 /content/en/     @istio/wg-docs-maintainers-english
+/content/en/blog @istio/steering-committee-maintainers @istio/steering-committee-members
```

```suggestion
/content/en/blog @istio/steering-committee 
```

I think the ones mentioned here dont exist?

---

Thanks, I was putting the structure incorrect, thought it can read the maintainers and members section under steering committee